app/controllers/leidinggevendenbeheer/bestuursfunctie/functionarissen/edit.js

```diff
@@ -0,0 +1,15 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class LeidinggevendenbeheerBestuursfunctieFunctionarissenEditController extends Controller {
+  @service router;
+
+  // TODO make sure filled in data is reset after cancelling
```

Comment:
won't the transition do this automatically? we didn't update any ember models and will fetch the instance again on reloading the route

Reply:
Ah yes with forms this is probably fine, in the legacy code this was partially changed in the model, so this needed to be reset. I'll remove this TODO.